doc/nrf/test_and_optimize/optimizing/memory.rst

```diff
@@ -256,6 +256,16 @@ Besides applying `General recommendations`_, you can also complete the following
   * :kconfig:option:`CONFIG_MAIN_STACK_SIZE`
   * :kconfig:option:`CONFIG_ISR_STACK_SIZE`
 
+.. _app_memory_wifi:
+
+Wi-Fi
+*****
+
+The current Wi-Fi® memory requirements are listed on the :ref:`ug_wifi_mem_req` page.
+
+In addition to applying `General recommendations`_, you can also see the :ref:`nRF7002dk_nRF5340_constrained_host` page to optimize the Wi-Fi part of your application.
```

Comment:
"wi-fi part..." I guess we can refer to it as the "Wi-Fi stack in your application"